### PR TITLE
Update update-api-with-oas.md

### PR DESCRIPTION
**Pull request by letzya**
Rephrased to use OpenAPI Spec instead of OAS definiton

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ In order to use the Gateway API you will need an API key for your Gateway and on
 
 ### Open Source
 
-### Tutorial: Update an API using an OAS API definition
+### Tutorial: Update Tyk OAS API definition with an updated OpenAPI Specification
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Perhaps "Update a Tyk OAS API with an updated OpenAPI Specification"?

I'm not sure that we need the word "definition" here.

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ weight: 6
 
 ### Introduction
 
-While developing APIs as developers, we often work with OAS definitions that are either generated from our codebase, or built ahead by the API designers. Tyk allows you to update/patch a Tyk API definition by providing only an OAS API definition.
+While developing APIs, as developers, we constantly need to update the OpenAPI Specification that is either generated from our codebase, or built using some API design tools. Tyk allows you to update the Tyk API definition with fresh copy of your OpenAPI Specification using PATCH HTTP method.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: 1) I would delete the word "some", as it's a big vague.
2) For consistency, could you change "the Tyk API definition" to "a Tyk OAS API"?
3) There's a missing "a" before "fresh"
4) I'd add "the" before PATCH, and I'd also put PATCH in `code` format

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 3]
```diff
@@ -23,15 +23,15 @@ In order to use the Gateway API you will need an API key for your Gateway and on
...
 
-#### Make sure you know your API secret
+#### Make sure you know your Tyk API secret
 
 Your Tyk Gateway API secret is stored in your `tyk.conf` file, the property is called `secret`. You will need to use this as a header called `x-tyk-authorization` to make calls to the Gateway API.
 
 #### Create an API
 
-Create a new API by sending a minimalistic Tyk OAS API Definition [https://bit.ly/39tnXgO](https://bit.ly/39tnXgO) to the Gateway API endpoint:
+For the sake of this tutorial, create a new API by sending this Tyk OAS API Definition [https://bit.ly/39tnXgO](https://bit.ly/39tnXgO) to the Gateway API endpoint (this is an example and contains the very minimun required fields):
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Typo: "minimun" should be "minimum" - and I'd put "of" after it, before "required"

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 4]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
 curl -H "x-tyk-authorization: {your-secret}" -s http://{your-tyk-host}:{port}/tyk/reload/group
 ```
 
-#### Update an API using just the OAS definition
+#### Update your OpenAPI Specification outside Tyk
 
-Tyk allows you to update a Tyk OAS API definition by providing only an OAS API definition.
+Now let's assume you made some change in your OpenAPI Specification using some tool outside Tyk's domain. It could be for example adding a new path, changing description or any other change.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Perhaps there's a missing "the" between "changing" and "description"?

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 5]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
 
-Let’s define a new path, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme which updates your Tyk OAS API Definition with the new OAS API definition by sending a `PATCH` request to your Tyk Gateway.
+In this case we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (The OpenAPI Specification is in the code snippet in the next section, just to avoid repeatition)
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Typo: repeatition -> repetition

I'd suggest that, instead of "In this case we added..." we say "In this example we will add..."

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 6]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
-#### Update an API using just the OAS definition
+#### Update your OpenAPI Specification outside Tyk
 
-Tyk allows you to update a Tyk OAS API definition by providing only an OAS API definition.
+Now let's assume you made some change in your OpenAPI Specification using some tool outside Tyk's domain. It could be for example adding a new path, changing description or any other change.
 
-Let’s define a new path, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme which updates your Tyk OAS API Definition with the new OAS API definition by sending a `PATCH` request to your Tyk Gateway.
+In this case we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (The OpenAPI Specification is in the code snippet in the next section, just to avoid repeatition) 
+
+#### Update the Tyk API definition using just your updated OpenAPI Specification
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Tyk OAS API

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 7]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
+
+Tyk allows you to update a Tyk OAS API definition by providing only your OpenAPI Specification using `PATH` request.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: 1) Again, I would remove the word "definition"
2) There's a typo: PATH -> PATCH
3) Again, I would add "the" before PATCH

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 8]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
+Tyk will update the OpenAPI Specification section in the Tyk OAS API dfintion.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Typo: dfintion
(though I would just delete the word :)

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 9]
```diff
@@ -217,14 +222,15 @@ If the command succeeds, you will see the following response, where key contains
...
-```.curl
+```curl
 curl -H "x-tyk-authorization: {your-secret}" -s http://{your-tyk-host}:{port}/tyk/reload/group
 ```
-#### Protect your API based on the OAS definition
 
-Previously you updated the Tyk OAS API Definition with a new OAS definition, which is describing a security mechanism. In order for your Tyk Gateway to pick that up, and start protecting API access, the authentication mechanism needs to be enabled within your Tyk configuration as well.
+#### Protect your API based on the OpenAPI Specification
+
+Previously you updated the Tyk OAS API Definition with a new OpenAPI Specification, which is describing a security mechanism. In order for Tyk Gateway to use this security mechanism and start protecting the API, the authentication mechanism needs to be *enabled* within Tyk section in the Tyk OAS API definiton.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: 1) I'd remove "defintion"
2) "In order to" or "In order for" is unnecessarily wordy: perhaps just say "For Tyk Gateway to use this security mechanism..."
3) missing "the" between "within" and "Tyk"
4) Typo: definiton -> definition

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 10]
```diff
@@ -328,7 +334,7 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
 ```
 #### Explicitly allow access to documented endpoints
 
-While updating a Tyk API OAS Definition using just the OAS API Definition, you can also give instructions to the Gateway to explicitly allow access just to paths that are documented in the OAS API Definition. For that, we have to pass the `allowList` query parameter together with our payload.
+While updating a Tyk API OAS Definition using just the OpenAPI Specification, you can also give instructions to Tyk Gateway to explicitly allow access just to paths that are documented in the OAS API Definition. For that, we have to pass the `allowList` query parameter together with our payload.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Perhaps reword this to:
You can instruct the Tyk Gateway to allow access only to paths (endpoints) explicitly documented in the Tyk OAS API using the allow list functionality. If you pass the `allowList` query parameter during the `PATCH` operation, the allow list will be automatically configured according to the OpenAPI Specification in the payload.

 --> this perhaps gives the opportunity to add a link (to here? https://tyk.io/docs/tyk-apis/tyk-gateway-api/api-definition-objects/ip-whitelisting/)

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 11]
```diff
@@ -734,9 +744,10 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
     }
 }
 ```
+
 #### What did you just do?
 
-You have demonstrated that by using the OAS API Definition, which can be either generated from your source code or created as part of design first approach, you can update or configure the Tyk OAS API Definition, with the `x-tyk-api-gateway` configuration.
+You have demonstrated that by using theOpenAPI specification, which can be either generated from your source code or created as part of design first approach, you can update and configure the Tyk OAS API Definition accordingly.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Missing a space in "the OpenAPI specification"

Perhaps "a" or "your" before "design first approach"?

As above, I'd remove the word "Definition".

**Review by andyo-tyk — COMMENTED** [review]
Comments added - some typos, some suggested changes.

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 2 above)
Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: 1. I wrote **some** since there are many tools in the market and wasn't sure if I should name a few or not

Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
While developing APIs, as developers, we constantly need to update the OpenAPI Specification that is either generated from our codebase or built using some API design tools. Tyk allows you to update the Tyk OAS API definition with a fresh copy of your OpenAPI Specification using the PATCH HTTP method.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 12]
```diff
@@ -13,7 +13,7 @@ weight: 6
 
 ### Introduction
 
-While developing APIs as developers, we often work with OAS definitions that are either generated from our codebase, or built ahead by the API designers. Tyk allows you to update/patch a Tyk API definition by providing only an OAS API definition.
+While developing APIs, as developers, we constantly need to update the OpenAPI Specification that is either generated from our codebase or built using some API design tools. Tyk allows you to update the Tyk OAS API definition with a fresh copy of your OpenAPI Specification using the PATCH HTTP method.
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: @andyo-tyk your message is gone, please re-review this bit. 
I used "some" since wasn't sure if I should name a few tools like swagger UI and stoplight or not. What do you think is best? essentially we want to say it's outside tyk's domain

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 1 above)
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
### Tutorial: Update a Tyk OAS API with an updated OpenAPI Specification
```

Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: I think we do since it's not an API, it's an API definition but I removed it

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 3 above)
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
For the sake of this tutorial, create a new API by sending this Tyk OAS API Definition [https://bit.ly/39tnXgO](https://bit.ly/39tnXgO) to the Gateway API endpoint (this is an example that contains the very minimal required fields):
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 4 above)
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Now let's assume you made some changes in your OpenAPI Specification using some tool outside Tyk's domain. It could be for example adding a new path, changing the description or any other change.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 5 above)
Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: spot on!

Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
In this example we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (The OpenAPI Specification is in the code snippet in the next section, just to avoid repetition) 
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 6 above)
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
#### Update the Tyk 
OAS API using just your updated OpenAPI Specification
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 7 above)
Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: 1. In the title we can be shorter but in the explanation, we should be accurate, why remove the word "definition"?

Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Tyk allows you to update a Tyk OAS API definition by providing only your OpenAPI Specification using the `PATCH` request.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 8 above)
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
As a result, Tyk will replace the existing OpenAPI Specification in the Tyk OAS API with the new OpenAPI Specification (from the payload).
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 9 above)
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Previously you updated the Tyk OAS API Definition with a new OpenAPI Specification, which is describing a security mechanism. For Tyk Gateway to use this security mechanism and start protecting the API, the authentication mechanism needs to be *enabled* within the Tyk section in the Tyk OAS API definition.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 10 above)
Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
You can instruct the Tyk Gateway to allow access only to paths (endpoints) explicitly documented in the OpenAPI Specification, using the allow list functionality. If you pass the `allowList` query parameter during the PATCH operation, the allow list will be automatically configured.
```

Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: 1. Changed it a bit as we were repeating ourselves.
2. This link shows fields of classic API def. Do we have OAS API def?

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 11 above)
Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
You have demonstrated that by using the OpenAPI Specification, which can be either generated from your source code or created as part of a design-first approach, you can update and configure the Tyk OAS API Definition accordingly.
```

Comment (letzya) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Just to explain about "definition":
1. It is really a definition, not an API, the API is the actual service. 
2. if it's not there, people could be mistaken with the Tyk APIs (which we also call "tyk oas" or "tyk OpenAPI specification")

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: On reflection, you're right, we do need something at the end of the sentence - I understand it's an OAS document (https://spec.openapis.org/oas/v3.1.0#openapi-document)... and I think we don't need it after Tyk OAS API, because we're actually talking about updating the API on the Gateway. 🤦 

```suggestion
### Tutorial: Update a Tyk OAS API with an updated OpenAPI Specification document 
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 2 above)
Comment (andyo-tyk) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Looks good. 👍

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Actually, I've done more research and this should be "OpenAPI document" (there's no such thing as an OpenAPI Specification document"

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Update an API with OAS"
+title: "Update existing Tyk API with changes in your OpenAPI definition"
 date: 2022-07-13
-tags: ["Tyk Tutorials", "Getting Started", "First API", "Tyk Cloud", "Tyk Self-Managed", "Tyk Open Source", "Updating an API with OAS"]
-description: "Updating an API with OAS"
+tags: ["Tyk Tutorials OpenAPI", "Getting Started OpenAPI", "First API OpenAPI", "Tyk Cloud OpenAPI", "Tyk Self-Managed OpenAPI", "Tyk Open Source OpenAPI", "Updating API with OAS", "Update OpenAPI definition", "Import an OpenAPI file to update an existing API definition"]
+description: "Updating Tyk OAS API definition following in your OpenAPI defintion"
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
description: "Updating Tyk OAS API following a change in your OpenAPI definition"
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 14]
```diff
@@ -23,15 +23,15 @@ In order to use the Gateway API you will need an API key for your Gateway and on
...
 
-#### Make sure you know your API secret
+#### Make sure you know your Tyk API secret
 
 Your Tyk Gateway API secret is stored in your `tyk.conf` file, the property is called `secret`. You will need to use this as a header called `x-tyk-authorization` to make calls to the Gateway API.
 
 #### Create an API
 
-Create a new API by sending a minimalistic Tyk OAS API Definition [https://bit.ly/39tnXgO](https://bit.ly/39tnXgO) to the Gateway API endpoint:
+For the sake of this tutorial, create a new API by sending this Tyk OAS API Definition [https://bit.ly/39tnXgO](https://bit.ly/39tnXgO) to the Gateway API endpoint (this is an example that contains the very minimal required fields):
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md:34]:
```suggestion
For the sake of this tutorial, create a new API by sending this [Tyk OAS API Document](https://bit.ly/39tnXgO) to the Gateway API endpoint (this is an example that contains the very minimal required fields):
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 15]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
 curl -H "x-tyk-authorization: {your-secret}" -s http://{your-tyk-host}:{port}/tyk/reload/group
 ```
 
-#### Update an API using just the OAS definition
+#### Update your OpenAPI definition
 
-Tyk allows you to update a Tyk OAS API definition by providing only an OAS API definition.
+Now let's assume you made some change in your OpenAPI definition (as mentioned above, from code or a tool, outside Tyk's domain). The change could be for  adding a new path, changing a description or anything that changes the definition.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Now let's assume you made a change in your OpenAPI definition (as mentioned above, from code or a tool, outside Tyk's domain). The change could be adding a new path, changing a description or anything that changes the definition of the API.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 16]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
 
-Let’s define a new path, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme which updates your Tyk OAS API Definition with the new OAS API definition by sending a `PATCH` request to your Tyk Gateway.
+In this example we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (The OpenAPI definition is in the code snippet in the next section, just to avoid repetition)
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
In this example we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (the OpenAPI definition is in the code snippet in the next section, just to avoid repetition) 
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 17]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
 
-Tyk allows you to update a Tyk OAS API definition by providing only an OAS API definition.
+Now let's assume you made some change in your OpenAPI definition (as mentioned above, from code or a tool, outside Tyk's domain). The change could be for  adding a new path, changing a description or anything that changes the definition.
 
-Let’s define a new path, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme which updates your Tyk OAS API Definition with the new OAS API definition by sending a `PATCH` request to your Tyk Gateway.
+In this example we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (The OpenAPI definition is in the code snippet in the next section, just to avoid repetition) 
+
+#### Update the Tyk API definition using just your updated OpenAPI definition
+
+Tyk allows you to update a Tyk OAS API definition by providing only your OpenAPI definition using `PATH` request.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
You can update your Tyk OAS API definition very simply - by providing your OpenAPI definition as an OpenAPI document with the `PATH` request.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 18]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
+Tyk will update the OpenAPI definition section in the Tyk OAS API dfintion.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Tyk will use the content of the OpenAPI document to update just the OpenAPI section in the Tyk OAS API definition.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 19]
```diff
@@ -217,14 +222,15 @@ If the command succeeds, you will see the following response, where key contains
...
-```.curl
+```curl
 curl -H "x-tyk-authorization: {your-secret}" -s http://{your-tyk-host}:{port}/tyk/reload/group
 ```
-#### Protect your API based on the OAS definition
 
-Previously you updated the Tyk OAS API Definition with a new OAS definition, which is describing a security mechanism. In order for your Tyk Gateway to pick that up, and start protecting API access, the authentication mechanism needs to be enabled within your Tyk configuration as well.
+#### Protect your API based on the OpenAPI definition
+
+Previously you updated the Tyk OAS API Definition with a new OpenAPI definition, which is describing a security mechanism. In order for Tyk Gateway to use this security mechanism and start protecting the API, the authentication mechanism needs to be *enabled* within Tyk section in the Tyk OAS API definiton.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Previously you updated the Tyk OAS API definition with a new OpenAPI definition, that describes a new security mechanism. In order for Tyk Gateway to start protecting the API using this authentication mechanism, it needs to be *enabled* within the Tyk section of the Tyk OAS API definition.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 20]
```diff
@@ -217,14 +222,15 @@ If the command succeeds, you will see the following response, where key contains
...
 
-For that, together with the `PATCH` request you just performed, you’re going to add the authentication query parameter, that tells Tyk to automatically enable authentication, based on the settings in the OAS definition.
+For that, together with the `PATCH` request you just performed, add the authentication query parameter `authentication=true`, that tells Tyk to automatically enable authentication, based on the settings in the OAS definition.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
To do this you would add the query parameter `authentication=true` to the `PATCH` request you just performed; this tells Tyk to automatically enable authentication, based on the settings in the OpenAPI definition.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 21]
```diff
@@ -328,7 +334,7 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
 ```
 #### Explicitly allow access to documented endpoints
 
-While updating a Tyk API OAS Definition using just the OAS API Definition, you can also give instructions to the Gateway to explicitly allow access just to paths that are documented in the OAS API Definition. For that, we have to pass the `allowList` query parameter together with our payload.
+While updating a Tyk API OAS Definition using just the OpenAPI definition, you can also give instructions to Tyk Gateway to explicitly allow access just to paths that are documented in the OAS API Definition. For that, we have to pass the `allowList` query parameter together with our payload.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
Tyk Gateway's allow list function explicitly allows access just to paths that are documented in the Tyk OAS API definition. You can enable this when updating a Tyk API OAS definition using the `PATCH` method by passing the `allowList` query parameter with the payload.
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 22]
```diff
@@ -734,9 +744,10 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
     }
 }
 ```
+
 #### What did you just do?
 
-You have demonstrated that by using the OAS API Definition, which can be either generated from your source code or created as part of design first approach, you can update or configure the Tyk OAS API Definition, with the `x-tyk-api-gateway` configuration.
+You have demonstrated that by using theOpenAPI definition, which can be either generated from your source code or created as part of design first approach, you can update and configure the Tyk OAS API Definition accordingly.
```
Comment (andyo-tyk) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
You have demonstrated that by using just the OpenAPI definition, which can be either generated from your source code or created as part of design first approach using other tools, you can update and configure an existing Tyk OAS API Definition.
```

**Review by andyo-tyk — CHANGES_REQUESTED** [review]
Some (hopefully final) tweaks to improve readability.

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 23]
```diff
@@ -99,11 +99,16 @@ Once you have created your API, you will need to either restart the Tyk Gateway,
...
 
-Tyk allows you to update a Tyk OAS API definition by providing only an OAS API definition.
+Now let's assume you made a change in your OpenAPI definition (as mentioned above, from code or a tool, outside Tyk's domain). The change could be adding a new path, changing a description or anything that changes the definition of the API.
 
-Let’s define a new path, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme which updates your Tyk OAS API Definition with the new OAS API definition by sending a `PATCH` request to your Tyk Gateway.
+In this example we added a new endpoint, `POST /pet`, with a schema that validates the payload it receives (`requestBody.content.application/json.schema`) and a new security scheme (the OpenAPI definition is in the code snippet in the next section, just to avoid repetition) 
+
+#### Update the Tyk API definition using just your updated OpenAPI definition
+
+You can update your Tyk OAS API definition very simply - by providing your OpenAPI definition as an OpenAPI document with the `PATH` request.
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
You can update your Tyk OAS API definition very simply - by providing your OpenAPI definition as an OpenAPI document with the `PATCH` request.
```

**Review by andyo-tyk — APPROVED** [review]
Looks great, thanks.
